remoteappmanager/static/js/admin/vue-components/toolkit/AdminLTEBox.js

```diff
@@ -0,0 +1,23 @@
+define([
+], function() {
+  "use strict";
+  
+  return {
+    props: {
+      title: {
+        type: String,
+        default: "Box"
+      }
+    },
+    template:
+    '  <div class="row">' +
+    '    <div class="col-md-12">' +
+    '      <div class="box">' +
+    '        <div class="box-header with-border"><slot name="header">{{title}}</slot></div>' +
+    '        <div class="box-body"><slot></slot></div>' +
```

Comment:
No, I keep it anonymous so everything that is between the component markers is embedded there.